build.sbt

```diff
@@ -4,7 +4,7 @@ name := "sbt-verify"
 
 organization := "uk.co.josephearl"
 
-version := "0.3.0-SNAPSHOT"
+version := "0.4.0"
```

Comment:
There already is a `v0.4.0` tag for a previous commit, so I think we need to bump it here to `0.4.1` and re-tag it as `v0.4.1`  after the new version is comitted.